spec/index.bs

```diff
@@ -71,6 +71,9 @@ spec: WEBGL-1; urlPrefix: https://www.khronos.org/registry/webgl/specs/latest/1.
         text: WebGLContextAttributes; url: WEBGLCONTEXTATTRIBUTES
     type: dfn
         text: WebGL Drawing Buffer; url: THE_DRAWING_BUFFER
+spec: Vulkan; urlPrefix: https://registry.khronos.org/vulkan/specs/1.3/html/
+    type: dfn
+        text: Scale Factor Operation, LOD Operation and Image Level Selection; url: chap16.html#textures-lod-and-scale-factor
```

Comment:
Adding the custom biblio works! But because it's not a bikeshed/W3C spec the link formatting is a bit of a pain.

 - `[[vulkan#textures-lod-and-scale-factor]]` Gives us a link text of **"Vulkan 1.3 § textures-lod-and-scale-factor"**
 - `[[vulkan#textures-lod-and-scale-factor|Scale Factor...]]` Gives us a link text of **"Scale Factor..."**

So we'd have to link it as `[[vulkan#textures-lod-and-scale-factor|Vulkan 1.3 § Scale Factor...]]` which I'm not particularly thrilled about.

Reply:
Ah of course, I forgot in addition to not knowing the chapter number it wouldn't know the title....

Looks like Webref is the place that indexes definitions, section headers, etc. Actually, they index WebGL already - maybe we don't even need the dfn hack for WebGL: https://github.com/w3c/webref/blob/main/ed/headings/webgl1.json
So maybe we could get Vulkan indexed there too, somehow?

Reply:
For the moment I've split the difference, because I _do_ want to use biblio for the reference:

`See [[vulkan#textures-lod-and-scale-factor|Scale Factor Operation, LOD Operation and Image Level Selection]] in the [[vulkan inline]] spec`

Which renders as:

**See [Scale Factor Operation, LOD Operation and Image Level Selection](https://registry.khronos.org/vulkan/specs/1.3/html/vkspec.html#textures-lod-and-scale-factor) in the [Vulkan 1.3](https://registry.khronos.org/vulkan/specs/1.3/html/vkspec.html) spec**

Reply:
Looks good! Didn't know about `inline`, that's handy.